src/main/resources/org/jenkinsci/plugins/github_branch_source/Messages.properties

```diff
@@ -52,6 +52,8 @@ BranchDiscoveryTrait.onlyPRs=Only branches that are also filed as PRs
 BranchDiscoveryTrait.allBranches=All branches
 BranchDiscoveryTrait.authorityDisplayName=Trust origin branches
 BranchDiscoveryTrait.displayName=Discover branches
+BranchOrTagAgeDiscoveryTrait.displayName=Discover branches or tags newer than X days
+BranchOrTagAgeDiscoveryTrait.authorityDisplayName=Trust branches or tags newer than X days
```

Comment:
Is this correct? (Sorry, for the multiple cycles of comments I thought I'd added this one yesterday, but hadn't clicked the button.)

Reply:
That is the location I changed. It is correct now. Any branches or tags not older than `X` days  are accepted.